internal/controller/bindata/networkfn-nad-host/00.networkfun-nad.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: sriov-net1
+  namespace: default
+  annotations:
+    k8s.v1.cni.cncf.io/resourceName: {{.ResourceName}}
+spec:
+  config: '{
+    "type": "dpu-cni",
+    "cniVersion": "0.4.0",
+    "name": "dpu-cni",
+    "ipam": {
+      "type": "host-local",
+      "subnet": "10.56.217.0/24"
```

Comment:
I don't know how useful this change is for the host is.

1. As you noticed "ipam" can be configured by the nad. The IP Address "10.56.217.0/24"" is not relevant for most if not all customer setups. Yes it is relevant for CDA, but we aren't building this for CDA.
2. Also  there are different types of IPAMs 
3. dpu-cni could configure parameters like MTU on the host.
4. In short, the reason why I only did the NAD for the DPU is that it is assumed the DPU's Network Functions will be created by the operator only. But the host NAD would be consumed by workload pods that are outside our control.

Reply:
host NAD will be created by the operator as a consequence of creating a DpuNetwork CR

Reply:
This means that the DpuNetwork CR must contain IPAM settings for the NAD in the future.